src/main/java/com/google/sps/model/TasksResponse.java

```diff
@@ -14,61 +14,21 @@
 
 package com.google.sps.model;
 
+import com.google.appengine.repackaged.com.google.common.collect.ImmutableList;
 import java.util.List;
+import lombok.Builder;
 
 /**
  * Contains the summary tasks information that should be passed to the client, as well as the
  * methods to generate these statistics.
  */
+@Builder
```

Comment:
https://github.com/googleinterns/step7-2020/pull/88#discussion_r459092523
> This constructor is just at the edge of being too complicated. If you were to add any more fields, I would recommend switching to a Builder pattern for initializing this object.
The concern here is that it would be very easy to get the order of the parameters wrong, and there is no way to see that at the call site. Unit tests should help catch bugs, but it's better when the code is self-documenting.

Reply:
`RESOLVED`